### DISCUSSION TITLE
Is it any way, how I can run waveform, but without playing?

### DISCUSSION THREAD

**Discussion by SolarGuy8** [katspaugh/wavesurfer.js#3485] [Q&A]
I have global player, and this player create an audio/video element, which handle song.

  `const element = process.browser ? document.createElement('video') : null;`

Can I just run waveform, but without track play? Just to see a progress, like in slider with dynamic value? 

P.S. I set `muted = false`, but I think, that it's not Okay and two players (waveform player and my global player) will make my page slow.

**Comment by katspaugh** [comment]
You can share the same audio element between your actual audio player and wavesurfer. Just pass an existing audio element in the `media` option.

Also see https://wavesurfer.xyz/examples/?react-global-player.js

**Reply by SolarGuy8** [reply]
Okay, thank you!
